src/main/java/com/googleinterns/zoomtube/data/TranscriptLine.java

```diff
@@ -24,46 +23,37 @@
 @GenerateTypeAdapter
 @AutoValue
 public abstract class TranscriptLine {
-  public static final String ENTITY_KIND = "TranscriptLine";
-  public static final String PROP_LECTURE = "lecture";
-  public static final String PROP_START = "start";
-  public static final String PROP_DURATION = "duration";
-  public static final String PROP_CONTENT = "content";
-  public static final String PROP_END = "end";
-
-  /**
-   * Creates a TranscriptLine object.
-   *
-   * @param transcriptKey The key for the transcript.
-   * @param lectureKey The key for the lecture.
-   * @param start The starting Date for the transcript line.
-   * @param duration The duration for the timestamp as a Date.
-   * @param end The ending Date for the transcript line.
-   * @param content The text content of the transcript line.
-   */
-  public static TranscriptLine create(
-      Key transcriptKey, Key lectureKey, Date start, Date duration, Date end, String content) {
-    return new AutoValue_TranscriptLine(transcriptKey, lectureKey, start, duration, end, content);
-  }
-
+  /** The key for the transcript. */
   public abstract Key transcriptKey();
+  /** The key for the lecture. */
```

Comment:
Maybe add a new line after each of these

Reply:
done!